### ISSUE TITLE
Mapping list item not working

### ISSUE THREAD

**Issue by AvidSeeker** [SidOfc/mkdx#178]
**OS type**: Unix

**Vim**: Neovim

**Neovim version**: 0.8.0
`vim -v`: 9.0

**Reproduce steps**:
1. Run nvim safe mode with mkdx installed
2. type "- first item<enter>"
3. type "1. first item<enter>"

**Expected**:
Auto insert "-" and "2." for second item.

**Actual**:
New line.

I checked this on vim, the dash worked, but "1." didn't.
On neovim, both don't work.

**Comment by AvidSeeker** [comment]
Polyglot is disabled.

**Comment by SidOfc** [comment]
Hello there!

As long as mkdx is loaded properly (by putting it in a pack directory for example. `:h packages`), everything is working fine for me in Neovim. For me it did not work in Vim initially but I figured out I also needed `filetype plugin on` in my `.vimrc` for it to work.

If that line is already present, then I've got a few questions:

1. How did you "install" `mkdx`?
2. What is the command you used to "run vim in safe mode with mkdx installed"?
3. What is the output of `:script` _after_ you `:edit some-test-file.md`?
